BPL-CBH LCCN

```diff
@@ -0,0 +1,89 @@
+'MacroName:BPLLCCN
+'MacroDescription:LC Call version for BPL
+   
+   'LC Call number
+'Created by Miriam Gloger 
+'Copy Call number from 050 to 099 Call number for Brooklyn Public Library CBH
+
+Option Explicit
+
+Sub Main
+   Dim CS as Object
+   On Error Resume Next
+   Set CS  = GetObject(,"Connex.Client")
+   On Error GoTo 0
+   If CS  Is Nothing Then
+      Set CS  = CreateObject("Connex.Client")
+   End If
+  
+   
+  Dim sLccall as String
```

Comment:
I know I'm not consistent myself and Connexion does not care, but I think it is a good practice to use upper case for OML/VB commands. `as` is a command.
`Dim sLccall As String`